man/mono.1

```diff
@@ -134,10 +134,58 @@ on demand by using the
 .I mono_install_load_aot_data_hook
 method.
 .TP
+.I direct-icalls
+.Sp
+When this option is specified, icalls (internal calls made from the
+standard library into the mono runtime code) are invoked directly
+instead of going through the operating system symbol lookup operation.
+This requires use of the
+.I static
+option.
+.TP
 .I direct-pinvoke
 .Sp
 When this option is specified, P/Invoke methods are invoked directly
 instead of going through the operating system symbol lookup operation.
+This requires use of the
+.I static
+option.
+.TP
+.I dwarfdebug
+Instructs the AOT compiler to emit DWARF debugging information. When
+used together with the nodebug option, only DWARF debugging
+information is emitted, but not the information that can be used at
+runtime.
+.TP
+.I full
+.Sp
+This compiles the code to run under the assumption all code is AOTed, IL is stripped and the JIT is unavailable at runtime. This feature is experimental and must run with a special "aot_full" BCL profile. This profile does not ship with the standard distribution of Mono, so you may need to build this profile yourself from the Mono source.
```

Comment:
You don't need the special aot_full BCL profile for that, e.g. the monotouch profile works with it as well. What counts is that whatever profile you use must not use stuff that doesn't work in a fully AOTed world without JIT.

Also I don't think we should call this experimental given that every Xamarin.iOS app uses this.